app/models/miq_report.rb

```diff
@@ -252,6 +252,24 @@ def group_description=(_group_description)
     # which does not exist in the MiqReport class
   end
 
+  def columns_for_sorting(columns)
+    columns || sortby || col_order
+  end
+
+  def validate_sorting_columns(columns)
+    validate_columns(columns_for_sorting(columns))
+  end
+
+  def validate_columns(sorting_columns)
+    sorting_columns.split(",").collect do |attr|
+      if col_order&.include?(attr)
+        attr
+      else
+        raise ArgumentError, "#{attr} is not a valid attribute for #{name}"
```

Comment:
once we do this, we'll have to add `_()` localization.
Is it possible to avoid this raise?

Reply:
> once we do this, we'll have to add _() localization.

I used N_() (based on discussion with @mzazrivec)

> Is it possible to avoid this raise?

I am not sure, I think that this is a way how we inform user of API request that something is wrong with API query.
Are you ok with this?